services/frontend/src/services/scatter.js

```diff
@@ -3,10 +3,10 @@ import ScatterEOS from 'scatterjs-plugin-eosjs2'
 
 const network = {
   blockchain: 'eos',
-  protocol: 'https',
-  host: 'nodes.get-scatter.com',
-  port: 443,
-  chainId: 'aca376f206b8fc25a6ed44dbdc66547c36c6c33e3a119ffbeaef943642f0e906'
+  protocol: 'http',
+  host: 'jungle2.cryptolions.io',
+  port: 80,
+  chainId: 'e70aaab8997e1dfce58fbfac80cbbb8fecec7b99cf982a9444273cbc64c41473'
```

Comment:
Do we want to merge these config changes to master? I applied something similar locally for dev purposes, but if we merge this it means that the login will be performed against Jungle Testnet.
@gaboesquivel, @danazkari can you give us your input on this?

Reply:
Yes, Jungle

Reply:
Perfect then, thanks.